frontend/src/libs/components/form/form.tsx

```diff
@@ -39,7 +73,7 @@ const Form = <T extends FieldValues = FieldValues>({
 
   const createInputs = (): JSX.Element[] => {
     return fields.map((field, index) => (
-      <Input {...field} control={control} errors={errors} key={index} />
+      <div key={index}>{renderField(field, control, errors)}</div>
```

Comment:
don't you have warnings in the console about setting index to the key?

Reply:
What's wrong? renderField returns either a dropdown or an input based on the type and is wrapped by <div key={index}>

Reply:
@denred 
https://reactpatterns.js.org/docs/indexes-as-a-key-is-an-anti-pattern/